dev_guide/limits.adoc

```diff
@@ -47,25 +105,39 @@ Per container, the following must hold true if specified:
 |`*Min*`
 |Min[resource] <= container.resources.requests[resource] (required) <= container/resources.limits[resource] (optional)
 
+If the configuration defines a `min` CPU, then the request value must be greater
+than the CPU value. A limit value does not need to be specified.
+
 |`*Max*`
 |container.resources.limits[resource] (required) <= Max[resource]
 
+If the configuration defines a `max` CPU, then you do not need to define a
+request value, but a limit value does need to be set that satisfies the maximum
+CPU constraint.
+
 |`*MaxLimitRequestRatio*`
 |MaxLimitRequestRatio[resource] <= ( container.resources.limits[resource] / container.resources.requests[resource])
 
+If a configuration defines a `maxLimitRequestRatio` value, then any new
+containers must have both a request and limit value. Additionally, the
```

Comment:
I think this is still confusing.

If a configuration defines a `maxLimitRequestRatio` value, then any new containers must specify both a request and a limit value.  The system will calculate a limit to request ratio by dividing the limit by the request.  For example, if a container has a limit for cpu of 500m and a request for cpu of 100m, then its limit to request ratio for cpu is 5.  The limit to request ratio for the container must be less than or equal to the conigured maxLimitRequestRatio.